tasks/fonts.js

```diff
@@ -1,10 +1,13 @@
 'use strict';
 
+const touch = require('gulp-touch-cmd');
```

Comment:
We're using https://www.npmjs.com/package/gulp-load-plugins to import the gulp plugins automatically.

The package should be available via  `$.touchCmd`.